### DISCUSSION TITLE
System.Net.Mail.SmtpException

### DISCUSSION THREAD

**Discussion by joao-sotero** [balta-io/forum#259] [Tira Dúvidas] [answered]
Estou no curso de Fundamentos do ASP.NET 6 e na parte de envio do email pelo sendgrid, não estar funcionando.
Estou utilizando meu email pessoal para enviar o email, o mesmo que utilizei para criar a conta no sendgrid.
![image](https://user-images.githubusercontent.com/62224433/217690375-5e59e017-e16a-464a-9bee-b450e0a42acc.png)
![image](https://user-images.githubusercontent.com/62224433/217690487-14d63eca-15a1-4c84-9bb4-848de867729a.png)

**Comment by joao-sotero** [comment, marked as answer]
Fiz a "Verified sender identity" que a exception indicou, o erro parou de acontecer mais o email não chega
![image](https://user-images.githubusercontent.com/62224433/217694717-1b054822-cea1-46e7-8599-ba1c0612fefe.png)

**Reply by andrebaltieri** [reply]
Bom dia, @joao-sotero, como vai?

As contas gratuitas nem sempre tem prioridade na entrega de E-mails, pode demorar um pouco.
Caso o problema persista, você pode abrir um chamado no SendGrid para ver o que aconteceu, ou mesmo criar outra conta para testar.

Lembrando que o SendGrid foi só um exemplo no curso. Existem outras ferramentas como o SendInBlue, MailChimp e OneSignal que tem envio de E-mail.

**Comment by mathfrance** [comment]
@joao-sotero fiz o mesmo, não erro mas não enviou, pra vc chegou depois ai?

**Reply by joao-sotero** [reply]
@mathfrance, não chegou.

**Reply by andrebaltieri** [reply]
Pessoal, tentem com outros E-mails, as vezes o Outlook bloqueia... o Gmail também.

**Reply by mathfrance** [reply]
@joao-sotero na área "Activity Feed" estava aparecendo uma tarja amarela dizendo que a minha conta estava sobre review e que eu deveria entrar em contato com o suporte, ao clickar ele pede mais algumas informações. Mandei ontem à noite, e hj de manhã foi liberada, entregou todas as minhas tentativas de envio que estavam pendentes e agora a mensagem que aparece é "Finish your account setup in order to select a paid plan and unlock the full potential of Twilio SendGrid.".

**Comment by Leandromenezes81** [comment]
Estou tendo um problema similar. Ao tentar enviar o e-mail, recebo o seguinte erro:

_Error processing 'variables' request. Unknown Error: 0x80131500_

![image](https://user-images.githubusercontent.com/76979829/222521530-6bdc738f-986f-4f77-b8ee-9dcb9afbf59f.png)

![image](https://user-images.githubusercontent.com/76979829/222522171-dbbd244f-062f-470b-9b0b-9ab4857331fe.png)
